src/lib/fetchServerAPI.ts

```diff
@@ -40,11 +40,15 @@ class FetchServerAPI {
     body: any,
     nextInit: RequestInit = {},
     customHeaders: { [key: string]: string } = {},
+    type: string = 'default',
   ): Promise<any> {
     const response = await fetch(`${this.baseURL}${endpoint}`, {
       method: 'POST',
-      headers: { ...this.headers, ...customHeaders },
-      body: JSON.stringify(body),
+      headers:
+        type === 'multipart'
+          ? { ...customHeaders }
+          : { ...this.headers, ...customHeaders },
+      body: type === 'multipart' ? body : JSON.stringify(body),
       ...nextInit,
     })
     return response
```

Comment:
```suggestion
    return response.json()
```
으로 수정해야 서버 반환하는 response body를 가져올 수 있습니다!

Reply:
반영하였습니다!